packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -69,11 +69,11 @@ export const getUserDetails = () =>
             dispatch(actions.getDetails.request());
 
             try {
-                const userDetails = await request('/rest/V1/customers/me', {
-                    method: 'GET'
+                const { data } = await fetchUserDetails({
+                    fetchPolicy: 'network-only'
```

Comment:
I'd leave a comment explaining why we use `network-only` here.

Reply:
Fixed by d9eff35